src/ng2-smart-table/components/cell/cell-types/cell-textarea.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Output, EventEmitter, Input } from '@angular/core';
+
+import { Cell } from '../../../lib/data-set/cell';
+import { DefaultCellType } from './default-cell-type';
+
+@Component({
+  selector: 'cell-type-textarea',
+  template: `
+    <textarea [ngClass]="inputClass"
+              class="form-control"
+              [(ngModel)]="cell.newValue"
+              [name]="cell.getColumn().id"
+              [disabled]="!cell.getColumn().isEditable"
+              (click)="onClick($event)"
```

Comment:
@damnko can you remove `compiled`, `demo-dist` and `dll` folders and files like *.d.ts, *.js, *.js.map and *.metadata.json from project directory. Then try `npm run build:aot`.